### PR TITLE
feat(betabot): Added /demo version

### PR DESCRIPTION
**Pull request by Jorropo**

### CODE REVIEW

**Review by clegirar — APPROVED** [review]
LGTM 👍

File: go/cmd/betabot/main.go [hunk 1]
```diff
@@ -434,12 +436,25 @@ Type /help when you need info about available test commands! 📖`
 
 func doStep2(ctx context.Context, _ *Conversation, bot *Bot, receivedMessage *bertymessenger.AppMessage_UserMessage, interaction *bertymessenger.Interaction, unlock func()) (bool, error) {
 	unlock()
-	if receivedMessage.GetBody() == "/help" {
+	switch strings.ToLower(receivedMessage.GetBody()) {
```
Comment (clegirar) [inline go/cmd/betabot/main.go:439]: For the unavailable commands maybe can we put a default switch case that say `This command is not already available` or something like that ?
Now for example, a `/demo group` message, betabot auto reply a random answer

Comment (Jorropo) [reply go/cmd/betabot/main.go:439]: Very good point, I'll do that :)

**Comment by clegirar** [comment]
LGTM 👍

**Pull request merged**